clean_data.py

```diff
@@ -91,54 +101,34 @@ def sii_from_audiogram(
     return sii.sii(ssl=ssl, nsl=nsl, hearing_threshold=critical_band_hl)
 
 
-def sii_from_df(df, ear='R'):
+def sii_from_df(df, ear):   
   freqs = [250, 500, 1000, 2000, 3000, 4000, 6000, 8000]
   names = [f'{ear}{f}' for f in freqs]
   values = df[names].values
   names_values = list(zip(names, values))
+  
+  """Extracts hearing thresholds from the specified frequencies, filters out the 
+  invalid data (NaN or infinite values), and computes SII only if there are 2
+  valid points, with default of any ear     #Changed from default of right ear - VMA 
+  """
   try:
     good_names_values = [nv for nv in names_values if np.isfinite(nv[1])]
     if len(good_names_values) > 2:
       freqs, values = zip(*good_names_values)
       freqs = [float(f[1:]) for f in freqs]
 
-      r_sii = sii_from_audiogram(values, freqs)
+      ear_sii = sii_from_audiogram(values, freqs)
     else:
-      r_sii = np.nan
+      ear_sii = np.nan
   except:
-    r_sii = np.nan
-  return r_sii
-
+    ear_sii = np.nan
+  return ear_sii
 
 def calculate_all_sii(df: pd.DataFrame):
   df['R_SII'] = df.apply(lambda df: sii_from_df(df, 'R'), axis=1)
   df['L_SII'] = df.apply(lambda df: sii_from_df(df, 'L'), axis=1)
   return df
 
-
```

Comment:
Just curious, why did this code go away?  I don't see it computed somewhere else.  Is it?